### PR TITLE
Blog App Add devise

### PR DESCRIPTION
**Pull request by Isabellanamutenda**
In this project I:
Remove current_user method in ApplicationController
Setup device which:

- Can register a new user.
- User logs in with a combination of email and password.
- Hashed passwords should be stored in the database.
- Ask for confirmation of email.
- Can reset password.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (rloterh) [inline app/models/user.rb]: - [x] As we require that an email is confirmed during new registration process, kindly ensure to enable _:confirmable_ in devise authentication

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Ashley, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (Isabellanamutenda) [reply app/models/user.rb]: Fixed

File: app/models/user.rb [hunk 2]
```diff
@@ -3,14 +3,14 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
```
Comment (rloterh) [inline app/models/user.rb:5]: This issue still needs further improvement:
- [x] _Confirmable_ is not enabled. Kindly add `:confirmable` to _devise_ on _line 4_.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Ashley, 

You did great on this project, and for making the changes requested in the previous review(s)! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments below this review. Please take your time to carefully improve aspects outlined in the previous code review, and carefully check to see that the implemented changes are working as intended._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 2 above)
Comment (Isabellanamutenda) [reply app/models/user.rb:5]: I have enabled it now

**Pull request merged**